### ISSUE TITLE
Gene and Tag <a> tags should contain hrefs to the corresponding pages

### ISSUE THREAD

**Issue by joeyAghion** [artsy/force#1061]
Noticed today that artworks' "related works" menus, which provide gene or tag labels for clicking, no longer link to the corresponding pages. That leaves gene and tag pages almost completely inaccessible, except from direct term search or the `/categories` page.

![](https://cl.ly/062d141H3r16/download/Screen%20Shot%202017-03-21%20at%2010.57.24%20AM.png)

This isn't just best-practice. The client-side UI enabled by these elements isn't accessible to Google's crawler, which instead sees the `?_escaped_fragment_=` [static] version of the page where these would be actual links that would build relevance, page-rank, etc. for the destination pages. We're even investing in outreach for those pages now, which can hardly get far if the pages aren't discoverable.

**Comment by dblock** [comment]
So these don't look like destination pages (anymore) and are just selectors for rails of works. I don't think there's anything to do here?

![related](https://cloud.githubusercontent.com/assets/542335/24718185/208e19e6-1a03-11e7-9eeb-e69d2b18055c.gif)

**Comment by joeyAghion** [comment]
I'd like to reopen this since I think there _is_ some benefit to making these links actual links. Currently we don't link to gene or tag pages from _anywhere_ except the `/categories` page. They continue to rank poorly (as a result, in my opinion).

These might operate like tabs, but they're highly contextual and artsy.net in fact has excellent general pages corresponding to each, so it could be valuable to make them real links _even if only for Google's crawler_.

**Comment by madeleineb** [comment]
I agree this is worth addressing. It used to be that when you click on any of those selectors that are gene names, above the rail of artworks there was a link to "Explore all works in [gene]". 

This would also display if that selector were a tag, eg. "Profile".
